style.css

```diff
@@ -1,10 +1,257 @@
-body {
-background-color: grey;
-font-family: "GothamPro";
+.body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  align-content: center;
 }
-
-.item__price {
-font-weight: bold;
+.container {
+  max-width: 1030px;
+  margin-left: 0;
 }
-
- 
+.header {
+  max-height: 96px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 20px;
+}
+.header-menu,
+.header-menu__right {
+  display: flex;
+  margin-top: 28px;
+  justify-content: space-between;
+}
+.header-menu__logo {
+  width: 55px;
+  height: 55px;
+  margin-right: 45px;
+  margin-top: 10px;
+}
+.header-menu__nav,
+.header-menu__right {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.header-menu__nav-link, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/step/pullRequest?prid=5998468)